### ISSUE TITLE
ShapeInferenceError in v0.3.1

### ISSUE THREAD

**Issue by xkszltl** [Microsoft/onnxruntime#796]
**Describe the bug**

This may be a regression in v0.3.1

We got the following error after rebuilding Ort with latest release tag (v0.3.0 -> v0.3.1).
There was similar discussion previously about this error message and it was fixed in v0.3.0.

```
:[ShapeInferenceError] Number of elements of input 'scales' must be same as rank of input 'X' and element type must be float.
" thrown in the test body.
```

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Win10
- ONNX Runtime installed from (source or binary): source
- ONNX Runtime version: v0.3.1
- Visual Studio version (if applicable): VS2019

**Comment by snnn** [comment]
Does it exist in our CI build log?

**Comment by xkszltl** [comment]
It is a new exception in the unit test of our code after we rebuild it with latest release tag today.

**Comment by snnn** [comment]
So, it's not in the CI? If it is not reproducible in CI, I wouldn't consider it is a bug.

**Comment by RyanUnderhill** [comment]
Do you think this is that same ShapeInference bug in Onnx I ran into last week? Where the float_data is used, but not the raw_data in the tensor proto inside of the ShapeInference function?

**Comment by easonliu** [comment]
> Do you think this is that same ShapeInference bug in Onnx I ran into last week? Where the float_data is used, but not the raw_data in the tensor proto inside of the ShapeInference function?

I believe it's the same bug. I'm inferencing the opet_v9 model which with upsampling op.

**Comment by snnn** [comment]
@faxu Should we put this into ONNX 1.5's task list?

**Comment by raymondxyang** [comment]
@xkszltl would mind providing a model producing this error? Thx

**Comment by xkszltl** [comment]
@raymondxyang 

The model contains custom ops so if won't help much.
@RyanUnderhill should have more details on this.

**Comment by raymondxyang** [comment]
@xkszltl 

hi @RyanUnderhill is out this week.. could you send more details on it or point me to the previous discussion/fix? We are about to release onnx 1.5 and thinking about fixing it before the release is out.

If i understand rightly this error should be about resize op right?

**Comment by raymondxyang** [comment]
PR sent https://github.com/onnx/onnx/pull/1969 . Will following up the tests with the described model.
